### ISSUE TITLE
Layout UI: be consistent with status messages.

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#1698]
When *cancel* saving changes to a layout, you get this message:

> Layout **changes** discarded.

When saving changes, you get:

> Layout **content** saved.

I think that it would be better with "Layout changes saved" or "Layout configuration saved".

**Comment by maxmeno** [comment]
Well, I was baffled when I realized that in BD - after opening a block through the cog wheel and editing its content - I had to save the block and _then_ save the layout where the block was inserted.
In this case I guess you're saving actual content with the layout.

**Comment by klonos** [comment]
> In this case I guess you're saving actual content with the layout.

Yes, but that message does not only show when saving custom block content, so it needs to be generic.

Think of actual nodes of content: we cannot say "Node body field saved" when for example the user has edited the publishing status or added the node to a menu. It would be inaccurate. Similar situation here, hence "Layout changes saved" is more suitable (and consistent with "Layout changes *discarded*").

**Comment by maxmeno** [comment]
And simply "Changes saved" / "Changes discarded" ?

**Comment by docwilmot** [comment]
I think this makes sense as it is, actually. If you cancel then all your changes are discarded. If you save, you may not have made any changes and it may be confusing to the user if he didnt make any changes to be told that changes were saved, so appropriate message is the current state of content has been saved. IMHO.

**Comment by klonos** [comment]
But that is the **layout** edit form. You are not editing **content**. You are editing layout/block **configuration**.
